### PR TITLE
Add Cxense as amp-analytics vendor

### PR DESCRIPTION
**Pull request by SteinHDanCxense**
See #3654 for additional information. This PR adds vendor configuration support for Cxense analytics.

### CODE REVIEW

**Comment by SteinHDanCxense** [comment]
I signed it!



I signed it on behalf of my company: Cxense ASA





Best regards,

Stein

--

*Stein Hardy Danielsen*

Chief Architect & co-founder



*Cxense ASA*

Email: stein.danielsen@cxense.com

Phone: +47 91 66 71 44

Skype: SteinHDan

www.cxense.com





*From:* googlebot [mailto:notifications@github.com]
*Sent:* søndag 19. juni 2016 15.16
*To:* ampproject/amphtml
*Cc:* SteinHDanCxense; Author
*Subject:* Re: [ampproject/amphtml] Add Cxense as amp-analytics vendor
(#3655)



Thanks for your pull request. It looks like this may be your first
contribution to a Google open source project. Before we can look at your
pull request, you'll need to sign a Contributor License Agreement (CLA).

📝 *Please visit https://cla.developers.google.com/
<https://cla.developers.google.com/> to sign.*

Once you've signed, please reply here (e.g. I signed it!) and we'll verify.
Thanks.
------------------------------

   - If you've already signed a CLA, it's possible we don't have your
   GitHub username or you're using a different email address. Check your
   existing CLA data <https://cla.developers.google.com/clas> and verify
   that your email is set on your git commits
   <https://help.github.com/articles/setting-your-email-in-git/>.
   - If you signed the CLA as a corporation, please let us know the
   company's name.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/ampproject/amphtml/pull/3655#issuecomment-226996759>,
or mute the thread
<https://github.com/notifications/unsubscribe/AQdTrna7P2xw3kxOBRMKfsG_7iZ4jangks5qNUENgaJpZM4I5KcI>
.

File: extensions/amp-analytics/0.1/vendors.js [hunk 1]
```diff
@@ -300,6 +300,31 @@ export const ANALYTICS_CONFIG = {
     },
   },
 
+  'cxense': {
+    'vars': {
```
Comment (avimehta) [inline extensions/amp-analytics/0.1/vendors.js]: remove this block. Empty blocks are not needed.

**Comment by avimehta** [comment]
just one comment. rest lgtm. The CLA bot was not able to detect your signing. Can you make sure that the email address that you authored the commits with is part of the group?

**Comment by SteinHDanCxense** [comment]
Hi Avi,

Thank you for helping us to get this support into the platform.

I've removed the empty code block.
The CLA seems to have corrected itself.

Thanks again!

Best regards,
Stein :-)

**Pull request merged**